Survey/Sources/Presentation/Modules/Login/LoginViewModel.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  LoginViewModel.swift
+//  Survey
+//
+//  Created by Khanh on 27/07/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Combine
+import SwiftUI
+
+struct LoginViewModel {}
+
+extension LoginViewModel: ViewModel {
+
+    func transform(_ input: Input) -> Output {
+        let errorTracker = ErrorTracker()
+        let activityTracker = ActivityTracker(false)
+        let output = Output()
+
+        errorTracker
+            .receive(on: RunLoop.main)
+            .map { AlertMessage(error: $0) }
+            .assign(to: \.alert, on: output)
+            .store(in: &output.cancelBag)
+
+        activityTracker
+            .receive(on: RunLoop.main)
+            .assign(to: \.isLoading, on: output)
+            .store(in: &output.cancelBag)
+
+        return output
+    }
+}
+
+// MARK: - Input & Output
+
+extension LoginViewModel {
+
+    struct Input {
+
+        let loadTrigger: Driver<Void>
```

Comment:
```suggestion
        let loginTrigger: Driver<Void>
```
I think this is clearer

Reply:
It's just redundant code too, I removed it 🥲 at https://github.com/nkhanh44/ic-survey-ios/commit/45ae841b35aa065645f32fb9c3e9b222f1e64540